source/JSONFaceView.mc

```diff
@@ -57,15 +59,17 @@ class JSONFaceView extends WatchUi.WatchFace {
         disconnectedString = Application.loadResource(Rez.Strings.disconnected);
     }
 
-    private function initAllowedFeatures(){
-        featureList = Application.loadResource(Rez.JsonData.features);
+    public function updateSettingProperties(){
+        lines = new [10];
```

Comment:
Same than in JSONFaceSetting with the hardcoded value (10)

Reply:
As described in another comment I have pushed some changes to improve this.